src/main/webapp/WEB-INF/web.xml

```diff
@@ -0,0 +1,17 @@
+<!DOCTYPE web-app PUBLIC
+ "-//Sun Microsystems, Inc.//DTD Web Application 2.3//EN"
+ "http://java.sun.com/dtd/web-app_2_3.dtd" >
+
+<web-app 
+    xmlns="http://java.sun.com/xml/ns/javaee"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="http://java.sun.com/xml/ns/javaee http://java.sun.com/xml/ns/javaee/web-app_2_5.xsd"
+    version="2.5">
+  <display-name>ROOT Mock Redirector</display-name>
+  
+  <welcome-file-list>
+    <welcome-file>index.html</welcome-file>
```

Comment:
This is one of the defaults, I think.  I wonder if a web.xml is actually needed for this?